CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-express ChangeLog
 
+## 3.3.0 -
```

Comment:
I expect us to include this new option along with a breaking change for the default `saveInitialized` here: https://github.com/digitalbazaar/bedrock-express/issues/17 ... so this will end up being 4.x.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-express/pull/21/commits/5ce54a9fb85909399bed10f0de8a1430500e5e85